Code-the-Dream-School/dd-prac-team1-front
# Search PRA-111

## Searching through the recipes
## Requirements

Using the navbar input, the user can search through the recipes' tags, names, ingredients.

## Notes

None

## Test Steps

1)You can type an ingredient, a tag, or a recipe name and see the relevant recipes on savedRecipes page.
2)If you are on addRecipe page, you can also type a tag, a name or a recipe ingredient, and get redirected to the savedRecipes page showing the relevant results.
3)If the name/tag/ingredient is not found, you can see the relevant message.

## UI/UX Outcome

<img width="1451" alt="Screen Shot 2023-08-09 at 19 35 35" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/6279e1d1-f779-41c1-b04d-278baa14c7f0">


## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
